flake-registry.json

```diff
@@ -298,116 +33,6 @@
         "repo": "nixpkgs",
         "type": "github"
       }
-    },
-    {
-      "from": {
-        "id": "templates",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "NixOS",
-        "repo": "templates",
-        "type": "github"
-      }
-    },
```

Comment:
This one is useful for `nix flake init --template templates#foobar`

Reply:
I do not consider templates as they are any useful at all. But that is probably another topic that has to be dealt with in nixos/nix…

Reply:
The Nix docs directly refer to the `templates` flake entry, e.g. it documents commands such as `nix flake show templates`  and `nix flake init -t templates#simpleContainer`. So this shouldn't be removed.

> I do not consider templates as they are any useful at all.

If you don't consider them useful, just don't use them, but also don't break them for other users.